ReactSkia/sdk/CurlNetworking.cpp

```diff
@@ -165,6 +164,15 @@ bool CurlNetworking::preparePostRequest(shared_ptr<CurlRequest> curlRequest, fol
   }
   
   curl_easy_setopt(curlRequest->handle, CURLOPT_POST, 1L);
+  /* get verbose debug output please */
+  curl_easy_setopt(curlRequest->handle, CURLOPT_POSTFIELDSIZE, (long)dataSize);
+  curl_easy_setopt(curlRequest->handle, CURLOPT_COPYPOSTFIELDS, dataPtr);
+  status = true;
+  return status;
+}
+
+void CurlNetworking::setHeaders(shared_ptr<CurlRequest> curlRequest, folly::dynamic headers) {
+  struct curl_slist *curlListRequestHeader = NULL;
```

Comment:
Better to add null check on curlListRequestHeader before setting CURLOPT_HTTPHEADER